packages/core/cms/faststore/content-types.json

```diff
@@ -186,6 +186,17 @@
                       }
                     }
                   }
+                },
+                "deliveryOptions": {
```

Comment:
idea: in another PR (in the feature branch 1.2) we could change the `deliverySettings` naming because it's only related to the Delivery Methods filter. We could do it in the 1.2 because `deliverySettings` was introduced there, so it would be better to change before merging to main. 

What do you think?

Reply:
Good point! We can explore this idea in this other PR. Leaving `regionalization` only with `inputField` and `idkPostalCodeLink`, right? And creating a `deliveryPromise` with `deliverySettings` (to be changed to `deliveryMethods`) and `deliveryOptions`.